test/evergreen.yml

```diff
@@ -3430,7 +3429,183 @@ tasks:
             # Record the start time, in seconds
             date +%s > ../time.txt
             # Perform the tests to get code coverage metrics for
+            . ../test/evergreen/find_cmake.sh
+            
             ${test_env_vars|} test/unittest/unittests
+
+            ${test_env_vars|} $CTEST -R '(ex_|lsm_workload|filesys|metadata_salvage|col_efficiency|dup_index_collator|compact_quick|test_checkpoint_4_mixed_timestamps|test_checkpoint_6_row_named_prepare|random_directio|scope|join_main_table_row|pad_byte_collator|meta_ckptlist_get_alloc)'
+
+            ${test_env_vars|} /opt/mongodbtoolchain/v4/bin/python3 ../test/suite/run.py test_rollback_to_stable03
```

Comment:
Could `/opt/mongodbtoolchain/v4/bin/python3` be replaced by `${python_binary|python3}`, the same as some lines in below?

Reply:
Good idea. That was copy/pasted from the original competition entry and should be updated.

Reply:
I've updated that now.